tests/test_gen2convert.py

```diff
@@ -116,6 +118,41 @@ def testRawFilters(self):
         rawI = self.butler.get("raw", instrument="HSC", exposure=903986, detector=16)
         self.assertEqual(rawI.getFilter().getName(), "i")
 
+    def testCuratedCalibrations(self):
+        """Test that defects, the camera, and the brighter-fatter kernel were
+        added to the Gen3 registry.
+        """
+        originInfo = DatasetOriginInfoDef(["raw", "calib"], [])
+        rowsWithCalibs = list(
+            self.butler.registry.selectDimensions(
+                originInfo, expression="",
+                neededDatasetTypes=["raw", "camera", "bfKernel", "defects"],
+                futureDatasetTypes=[],
+            )
+        )
+        rowsWithoutCalibs = list(
+            self.butler.registry.selectDimensions(
+                originInfo, expression="",
+                neededDatasetTypes=["raw"],
+                futureDatasetTypes=[],
+            )
+        )
+        self.assertGreater(len(rowsWithoutCalibs), 0)
+        self.assertEqual(len(rowsWithCalibs), len(rowsWithoutCalibs))
```

Comment:
This is just ignorance on my part, but I'm surprised that the two sets should be the same.

Maybe this is a misunderstanding from me.  The name of the method seems like it is trying to select the columns that represent the spanning set of dimensions that describe the set of datasets, but maybe that's not actually what it does.

Reply:
It's actually doing something a bit more complex - `selectDimensions` iterates over the data IDs that are jointly valid for all of the given datasetTypes, so in essence we're comparing the total number of raws in the database with the number of raws that have a `camera`, a `bfKernel`, and a set of `defects`.  I'll add some comments to that effect.

Reply:
OK, great, and thanks.